backend/docker-compose.yml

```
@@ -0,0 +1,52 @@
+    depends_on:
+      db:
+        condition: service_healthy
+    networks:
+      - app_network
+
+networks:
+  app_network:
+    driver: bridge
+
+volumes:
+  postgres_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)